homeassistant/components/binary_max_sensor/sensor.py

```diff
@@ -0,0 +1,161 @@
+"""Support for displaying the minimal and the maximal value."""
+import logging
+
+import voluptuous as vol
+
+import homeassistant.helpers.config_validation as cv
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+from homeassistant.const import (
+    CONF_NAME,
+    STATE_UNKNOWN,
+    STATE_UNAVAILABLE,
+    ATTR_UNIT_OF_MEASUREMENT,
+)
+from homeassistant.core import callback
+from homeassistant.helpers.entity import Entity
+from homeassistant.helpers.event import async_track_state_change
+
+_LOGGER = logging.getLogger(__name__)
+
+ATTR_MAX_VALUE = "max_value"
+ATTR_COUNT_SENSORS = "count_sensors"
+
+ATTR_TO_PROPERTY = [ATTR_COUNT_SENSORS]
+
+CONF_ENTITY_IDS = "entity_ids"
+CONF_MAX_VALUE = "max_value"
+
+ICON = "mdi:calculator"
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
```

Comment:
We no longer allow integrations to use a platform config section. See https://github.com/home-assistant/architecture/blob/master/adr/0007-integration-config-yaml-structure.md.

Please refactor this integration to have the config schema in the base integration module and the config yaml section under the top level integration domain key.